src/helper-modules/bpmn-helper/src/getters.js

```diff
@@ -899,8 +982,8 @@ function getPerformersFromElement(element) {
 
 /**
  * Parses ISO Duration String to number of years, months, days, hours, minutes and seconds
- * @param {String} isoDuration
- * @returns {Object} Object with number of years, months, days, hours, minutes and seconds
+ * @param {string} isoDuration
+ * @returns {{years: number, months: number, days: number, hours: number, minutes: number, seconds: number}} Object with number of years, months, days, hours, minutes and seconds
```

Comment:
Aren't the values all optional/null if some of the conditionals bodies in the function are not entered?